Common/Deployment/DeploymentLib.ps1

```diff
@@ -234,6 +234,7 @@ function ValidateResourceName()
         "microsoft.documentdb/databaseaccounts"
         {
             $resourceUrl = $global:docdbSuffix
+            $resourceBaseName = $resourceBaseName.Substring(0, [System.Math]::Min(19, $resourceBaseName.Length))
```

Comment:
Please put a comment here to explain why we have this fix, and how 19 comes out.